### PR TITLE
misst a curly bracket in the appsettings key AdminProtection

### PR DESCRIPTION
**Pull request by masv-scb**
misst a curly bracket in the appsettings.release

### CODE REVIEW

File: PxWeb/appsettings.Release.json [hunk 1]
```diff
@@ -40,8 +40,9 @@
     },
     "AdminProtection": {
       "IpWhitelist": [ "#{ADMIN_IPCONFIG}#" ],
-      "AdminKey": "#{API_ADMIN_KEY}#"
-    },
+      "AdminKey": [ "#{API_ADMIN_KEY}#" ]
```
Comment (likp) [inline PxWeb/appsettings.Release.json:43]: AdminKey is a string so there should not be any square brackets

**Review by likp — CHANGES_REQUESTED** [review]
The AdminProtection settings is under PxApiConfiguration although it should be located directly under the root. Compare to appsettings.json

File: PxWeb/appsettings.Release.json (hunk 1 above)
Comment (masv-scb) [reply PxWeb/appsettings.Release.json]: have now adjusted this

**Review by MikaelNordberg — APPROVED** [review]
Looks good

File: PxWeb/appsettings.Release.json (hunk 1 above)
Comment (likp) [reply PxWeb/appsettings.Release.json]: looks good now.

**Review by likp — APPROVED** [review]

**Pull request merged**